storybook/CMakeLists.txt

```diff
@@ -105,7 +109,7 @@ target_compile_definitions(QmlTests PRIVATE
         STATUSQ_MODULE_IMPORT_PATH="${STATUSQ_MODULE_IMPORT_PATH}"
         QUICK_TEST_SOURCE_DIR="${CMAKE_CURRENT_SOURCE_DIR}/qmlTests")
 target_link_libraries(QmlTests PRIVATE Qt5::QuickTest Qt5::Qml ${PROJECT_LIB} SortFilterProxyModel)
-add_test(NAME QmlTests COMMAND QmlTests)
+add_test(NAME QmlTests COMMAND QmlTests -platform offscreen)
```

Comment:
This will work on Linux only afaik

Reply:
On macOs seems to work the same way it works on linux. I see the same crash from the screen recording. But it's not crashing anymore if I remove `-platform offscreen`.

Reply:
It's already fixed in https://github.com/status-im/status-desktop/pull/12409 and rebased so it's no longer crashing :)

Reply:
It works on Linux and Mac, on Win the format is the same from what I see in source code (but I didn't check in practice).